packages/telegram-cloud-storage-stamper/README.md

```diff
@@ -0,0 +1,195 @@
+
+The Telegram Cloud Storage Stamper will, by default, store the API key used for signing in Telegram Cloud Storage under the key `TURNKEY_API_KEY`. A Cloud Storage "key" is the index under which a value is stored in Telegram Cloud Storage. This can be changed when using the `.create()` or `.setSigningKey()` functions. An API key is stored within Cloud Storage as a JSON string of the following object:
+```
+{
+  apiPublicKey: "compressedApiPublicKeyHex",
+  apiPrivateKey: "apiPrivateKeyHex",
+}
+```
+
+#### Argument Usage
+
+The `.create()` and `.setSigningKey()` functions take one of the following 4 sets of arguments:
+- No arguments: Use an API key at the default location within Telegram Cloud Storage `TURNKEY_API_KEY` and set that as the signing key
+- Just an API key: Store the passed in API key and at the default Telegram Cloud Storage location and set that as the signing key
+- Just a Cloud Storage key: Use an API key stored at the specified Telegram Cloud Storage key location and set that as the signing key
+- Both an API key and a Cloud Storage key: Store the passed API key at the specified Telegram Cloud Storage key location and set that as the signing key
+  
+The `.getAPIKey()` and `.setAPIKey()` functions operate in a similar manner taking an optional `key` parameter that will be used to `get` or `set` the API key at that location if it is passed, or at the default location if it is not passed.
+
+### Usage
+Insert a new api key into Telegram Cloud Storage at the default API key location
+
+```ts
+import { TelegramCloudStorageStamper } from "@turnkey/telegram-cloud-storage-stamper";
+import { TurnkeyBrowserClient, TurnkeySDKClientConfig } from "@turnkey/sdk-browser";
+
+// create a new Telegram Cloud Storage Stamper
+const stamper = await TelegramCloudStorageStamper.create({
+  apiPublicKey: "...",
+  apiPrivateKey: "...",
+});
+
+if (!stamper) {
+  // There was a problem initializing the stamper
+}
+
+// use the stamper in the client config
+const browserConfig: TurnkeySDKClientConfig = {
+  stamper: stamper,
+  apiBaseUrl: "https://api.turnkey.com",
+  organizationId: <ORGANIZATION_ID>,
+};
+
+// create a TurnkeyClient with the initialized Telegram Cloud Storage Stamper
+const client = new TurnkeyBrowserClient(browserConfig);
+
+// make a request with the client
+const whoamiResponse = await client.getWhoami({
+  organizationId: <ORGANIZATION_ID>,
+});
+```
+
+Use an existing key that has been previously stored in Telegram Cloud Storage at the default API key location key location
+
+```ts
+import { TelegramCloudStorageStamper } from "@turnkey/telegram-cloud-storage-stamper";
```

Comment:
thoughts on calling this just `@turnkey/telegram-stamper`? Would that be pigeonholing too much? i.e. are there other types of telegram stampers that we could foresee having some day in the future?

Reply:
Had this discussion with @r-n-o earlier. It was originally telegram-stamper. There are talks of including Login with Telegram outside of a telegram mini app context. If we want to create a specialized stamper for that, this one will not work because there will not be access to TG cloud storage in a typical web app context so it is possible there is a different telegram stamper that could be developed. But its possible wed just advise users to utilize one our existing stampers if they are not in a telegram context